src/detect-parse.c

```diff
@@ -4229,6 +4229,38 @@ static int SigParseBidirWithSameSrcAndDest02(void)
     PASS;
 }
 
+static int SigParseTestActionReject(void)
+{
+    DetectEngineCtx *de_ctx = DetectEngineCtxInit();
+    FAIL_IF_NULL(de_ctx);
+
+    Signature *sig = SigInit(
```

Comment:
new code should use DetectEngineAppendSig

Reply:
When one forgets to learn from past mistakes x____x